src/mito2/src/compaction/twcs.rs

```diff
@@ -207,10 +211,13 @@ impl Window {
     /// Creates a new [Window] with given file.
     fn new_with_file(file: FileHandle) -> Self {
         let (start, end) = file.time_range();
+        let files = [(file.meta_ref().sequence, FileGroup::new_with_file(file))]
```

Comment:
We can use `HashMap::from` to convert an array into a HashMap.

Reply:
fixed in fixed in 3c4dd239e7